CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- **BREAKING:**: Event and initiator context value must be null or scalar.
```

Comment:
Not sure if this needs to be Breaking.

Code that passed something else than null or a scalar resulted in an error before this change too. It was just a backend error instead of a client error. This basically "just moves" the error from the backend to the frontend, for convenience. The change doesn't break working code. It breaks broken code in another way. :)

Reply:
Since the backend is vendor specific and not provided/descibed in this library, I'd vote for a breaking change because of the changed behaviour in _this_ library.